public/app/features/plugins/importPanelPlugin.ts

```diff
@@ -28,6 +28,14 @@ export function importPanelPlugin(id: string): Promise<PanelPlugin> {
   return promiseCache[id];
 }
 
+export function hasPanelPlugin(id: string): boolean {
+  return !!getPanelPluginMeta(id);
+}
+
+export function getPanelPluginMeta(id: string): PanelPluginMeta {
```

Comment:
The same code is used above to get meta in `importPanelPlugin` so maybe we could reuse it there?